dsew_community_profile/params.json.template

```diff
@@ -11,7 +11,11 @@
     "export_signals": [
       "confirmed covid-19 admissions",
       "total",
-      "positivity"
+      "positivity",
+      "doses administered",
+      "booster doses administered",
+      "fully vaccinated",
+      "booster dose since"
```

Comment:
Please add the full output names of any smoothed signals in `validation:dynamic:smoothed_signals` below, and add smoothed signals to the same spot and the export signal names to ansible template params.